client/src/featureform/serving.py

```diff
@@ -785,7 +785,7 @@ def label(self):
         return [self._label]
 
     def to_numpy(self):
-        return self._row()
+        return self._row
```

Comment:
Does these test directly test this change?

Reply:
the test added makes sure that `to_numpy()` returns a numpy array.